pkg/sentry/server/server.go

```diff
@@ -125,23 +141,25 @@ func (s *server) signCertificate(ctx context.Context, req *sentryv1pb.SignCertif
 	// daprd->daprd connections would fail. This is no longer the case since we
 	// now match with SPIFFE URI SAN, but we need to keep this here for backwards
 	// compatibility. Remove after v1.12.
-	var dns []string
+	var dns string
 	switch {
 	case req.Namespace == security.CurrentNamespace() && req.Id == "dapr-injector":
-		dns = []string{fmt.Sprintf("dapr-sidecar-injector.%s.svc", req.Namespace)}
+		dns = fmt.Sprintf("dapr-sidecar-injector.%s.svc", req.Namespace)
 	case req.Namespace == security.CurrentNamespace() && req.Id == "dapr-operator":
-		dns = []string{fmt.Sprintf("dapr-webhook.%s.svc", req.Namespace)}
+		dns = fmt.Sprintf("dapr-webhook.%s.svc", req.Namespace)
 	default:
-		dns = []string{fmt.Sprintf("%s.%s.svc.cluster.local", req.Id, req.Namespace)}
+		dns = fmt.Sprintf("%s.%s.svc.cluster.local", req.Id, req.Namespace)
 	}
 
+	log.Debugf("Processing SignCertificate requests for %s (validator: %s)", dns, validator.String())
+
 	chain, err := s.ca.SignIdentity(ctx, &ca.SignRequest{
 		PublicKey:          csr.PublicKey,
 		SignatureAlgorithm: csr.SignatureAlgorithm,
 		TrustDomain:        trustDomain.String(),
 		Namespace:          req.Namespace,
 		AppID:              req.Id,
-		DNS:                dns,
+		DNS:                []string{dns},
```

Comment:
Can you please leave this as it was?

The default switch on line 151 is only there for backwards compat, and will be removed in future meaning that an empty list is going to be passed to `SignIdentity` and so we are going to have to change it back anyway.

Reply:
Ok I understand.

I did this primarily for logging reasons, but I can move the log inside the switch.